libcontainerd/container_windows.go

```diff
@@ -20,6 +21,8 @@ type container struct {
 	// but can be called from the RestartManager context which does not
 	// otherwise have access to the Spec
 	ociSpec Spec
+
+	stopped sync.WaitGroup
```

Comment:
In this particular case, it's better to use a channel. Channel will allow you also to wait in select, which is a good idea too.